src/jit/emit.h

```diff
@@ -794,7 +774,7 @@ class emitter
 // Non-DEBUG sizes:
 //   x86/amd64/arm/arm64: 64 bits
 // DEBUG sizes (includes one pointer):
-//   x86:   2 DWORDs, 64 bits
+//   x86:   2 DWORDs, 96 bits
```

Comment:
I guess technically 3 DWORDS. Not sure why we need to list bits here, since we're trying to list fully aligned data sizes.

Reply:
> Not sure why we need to list bits here, since we're trying to list fully aligned data sizes.

Hah, there were so many bits around that I failed to see the DWORDs as well.

IMO it would be better to just put all data members in one place, under a single if/elif/endif. Then you would not need to summarize the layout in comments at various points.

Oh well, at least on Windows I can rely on dumping the data structures from PDB to figure out the layout.